cmd/ztest/ztest.c

```diff
@@ -709,6 +739,7 @@ usage(boolean_t requested)
 	    "\t[-o variable=value] ... set global variable to an unsigned\n"
 	    "\t    32-bit integer value\n"
 	    "\t[-G dump zfs_dbgmsg buffer before exiting due to an error\n"
+	    "\t[-X off raidz_expand test, killing at off bytes into reflow\n"
```

Comment:
```suggestion
	    "\t[-X off] raidz_expand test, killing at off bytes into reflow\n"
```
The above line has a similar typo, should be `[-G]`

Reply:
Will fix, I just copied the typo above, will fix that one too